src/pipdeptree/__init__.py

```diff
@@ -997,6 +1000,15 @@ def get_parser():
             " ignore this argument."
         ),
     )
+    parser.add_argument(
+        "--encoding",
+        dest="encoding_type",
+        default=None,
```

Comment:
Shouldn't thje default here be sys.stdout.encoding.lower(), because that's what we're defaulting to and render_text should not take an optional arg, should be mandatory.